cypress/integration/dashboard_spec.js

```diff
@@ -0,0 +1,40 @@
+describe('Dashboard User Flows', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3001');
+  });
+
+  it('Should display movie poster and ratings on individual movie cards on page load', () => {
+    cy.intercept('GET', 'https://rancid-tomatillos.herokuapp.com/api/v2/movies', {
+      statusCode: 201,
+      body: {
+         movies: [ 
+          {
+            average_rating: 6.142857142857143,
+            backdrop_path: "https://image.tmdb.org/t/p/original//pq0JSpwyT2URytdFG0euztQPAyR.jpg",
+            id: 694919,
+            poster_path: "https://upload.wikimedia.org/wikipedia/commons/6/6e/Golde33443.jpg",
```

Comment:
cute puppy pic :)

Reply:
We might want to add a second puppy pic to our second card to make sure the data is somewhat different than our actual data.